ravenframework/Models/ROM.py

```diff
@@ -172,6 +172,35 @@ def applyRunInfo(self, runInfo):
     """
     self.numThreads = runInfo.get('NumThreads', 1)
 
+  def _manipulateInput(self,dataIn):
```

Comment:
Can you remove this method from the ROM class? I think it is not needed. @mandd 
The check has been performed in createNewInput

Reply:
good point, removed